packages/lambda-tiler/src/util/config.loader.ts

```diff
@@ -46,10 +46,14 @@ export class ConfigLoader {
 
     req.set('config', configLocation);
     req.timer.start('config:load');
-    return CachedConfig.get(configLocation).then((f) => {
-      req.timer.end('config:load');
-      if (f == null) throw new LambdaHttpResponse(404, `Config not found at ${configLocation}`);
-      return f;
-    });
+    return CachedConfig.get(configLocation)
+      .then((f) => {
+        req.timer.end('config:load');
+        if (f == null) throw new LambdaHttpResponse(404, `Config not found at ${configLocation}`);
+        return f;
+      })
+      .catch((e) => {
+        throw new LambdaHttpResponse(e.status, e.statusDescription);
```

Comment:
this seems risky as `e` could be anything... if it was a string this response would be undefined, undefined which may break other things?

Reply:
Shall we both throw the same error. 400 with `Invalid config location at ${configLocation}`

Reply:
I think this catch is risky, we should handle the permission denied when we are actually reading the file